sdk/ai/azure-ai-resources/azure/ai/resources/operations/_connection_operations.py

```diff
@@ -60,14 +60,14 @@ def get(self, name: str, **kwargs) -> BaseConnection:
 
         # It's by design that both API and V2 SDK don't include the secrets from API response, the following
         # code fills the gap
+        # TODO do we still want this?
```

Comment:
Revert this?

Reply:
Yes, as per our discussion about this behavior working again.